lib/web/desktop.go

```diff
@@ -333,6 +336,21 @@ func proxyWebsocketConn(ws *websocket.Conn, wds net.Conn) error {
 		wds.Close()
 	}
 
+	sendTDPNotification := func(ws *websocket.Conn, err error, fatal bool) error {
+		var severity tdp.Severity
+		if fatal {
+			severity = tdp.SeverityError
+		} else {
+			severity = tdp.SeverityWarning
+		}
```

Comment:
```suggestion
		severity := tdp.SeverityError
		if !fatal {
			severity = tdp.SeverityWarning
		}
```

Pick a default so we can kill the else.